### PR TITLE
Travel survey - Johan Löfgren

### PR DESCRIPTION
**Pull request by johlofg**
Link to my deployed project:
https://travel-survey-johanlofgren.netlify.app/

My reflections on how this week's project turned out:
I´m happy with the resault of this project. Reaced the goals I was planning for and had a good understanding all the time. 

Things I'd like to have clarified or explained in more detail:
I have a range slider component, planed to implement it first, but just couldn´t decide how I wanted to use it. 
I will maybe implement it later.

### CODE REVIEW

File: code/src/Components/Summary.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+// This function decides if the summary text should have the form of a ice block regarding the choice of type of trip(radiobtn) 
+  let changeClass = "form-wrapper"
+  if (props.radioValue === "Winter") {
+    // eslint-disable-next-line
+    {changeClass = "form-wrapper ice"}
+  }
+
+  return (
+    <div className="summary" tabindex="0">
+      {/* This decides if the sun should be displayed or not in summary regarding to choice of typ of trip (radioBtn) */}
```
Comment (AsaHildebrand) [inline code/src/Components/Summary.js:13]: Good job with the explaining comments!

File: code/src/Components/TextInput.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
```
Comment (AsaHildebrand) [inline code/src/Components/TextInput.js:2]: Maby only have one empty line between import and export!

File: code/src/Components/TextInput.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+export const TextInput = (props) => {
+  
+
```
Comment (AsaHildebrand) [inline code/src/Components/TextInput.js:6]: No empty lines needed :)

File: code/src/Components/TextInput.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  return (
+  <div className="text-input" tabindex="0">
+    <label htmlFor="name-input">What´s your name: </label>
+    <input 
+      id="name-input"
+      type="text"
+      value={props.name}
+      onChange={props.onNameChange}
+      />
+      </div>
```
Comment (AsaHildebrand) [inline code/src/Components/TextInput.js:16]: Use indentations a bit diffrent, like you did in summary.js 👍

File: code/src/Components/TextWrapper.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+export const TextWrapper = () => {
+  return (
+    <div tabindex="0" className="text-wrapper">
+            <h1>Hi!</h1>
+            <h2>Welcome to this short travel survey!</h2>
+            <p>Hopefully all the countrys will soon open up and we can start living again.</p>
+            <p>For alot of people living is travling, how do you like to travel?</p>
+            <p>Please take a minute to fill out this survey.</p> 
+          </div>
```
Comment (AsaHildebrand) [inline code/src/Components/TextWrapper.js:11]: Same here with the indentations!

File: code/src/Components/DropDown.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+export const DropDown = (props) => {
+
```
Comment (AsaHildebrand) [inline code/src/Components/DropDown.js:4]: No empty line is needed here :)

File: code/src/Components/RadioButton.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+export const RadioButton = (props) => {  
+
```
Comment (AsaHildebrand) [inline code/src/Components/RadioButton.js:4]: No empty line needed.

File: code/src/Components/RangeSlider.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+// This one aint being used right now. Can be implementen in next version. Needs to add hook and function then
```
Comment (AsaHildebrand) [inline code/src/Components/RangeSlider.js:3]: Good with this explanation, hope you find a way to use it later :)

File: code/src/index.css [hunk 9]
```diff
@@ -11,3 +16,86 @@ code {
...
+  cursor: pointer;
+}
+
+@-webkit-keyframes shine {
+  0% {box-shadow: 0 0 70px 0 rgba(209, 155, 5, 0.863);}     
+  100% {box-shadow: 0 0 60px 0 rgba(213, 235, 10, 0.637);}   
+}
+@keyframes shine {
+  0% {box-shadow: 0 0 70px 0 rgba(209, 155, 5, 0.863);}  
+  100% {box-shadow: 0 0 50px 0 rgba(227, 252, 2, 0.637);}
```
Comment (AsaHildebrand) [inline code/src/index.css:83]: Nice effects with the sun!!

File: code/src/index.css [hunk 10]
```diff
@@ -11,3 +16,86 @@ code {
...
+  -webkit-backdrop-filter: blur( 20px );  
+  border: 1px solid rgba( 255, 255, 255, 0.18 );
+}
+
+.sun {
+  background-color: rgb(255, 255, 0);
+  width: 300px;
+  height: 300px;
+  border-radius: 50%;
+  animation: shine 1.5s infinite alternate;
```
Comment (AsaHildebrand) [inline code/src/index.css:99]: Nice effect! :)

**Review by AsaHildebrand — APPROVED** [review]
Very nice work with this project, I like the idea of the sun and the ice, depending on the answer! 
Nice that you have comments for the code that needs some more explaining. Good job! 👍

**Review by KarolCislowski — APPROVED** [review]
Great job!
Maybe for the next step, you will try to add some more exotic types of inputs? 🍹

**Pull request closed without merging**